tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -115,9 +121,19 @@ class Staging(CloudGov):
 
     ALLOWED_HOSTS = ['tdp-backend-staging.app.cloud.gov']
 
+    LOGIN_GOV_CLIENT_ID = os.getenv(
+        'OIDC_RP_CLIENT_ID',
+        'urn:gov:gsa:openidconnect.profiles:sp:sso:hhs:tanf-proto-staging'
+    )
+
 
 class Production(CloudGov):
     """Settings for applications deployed in the Cloud.gov production space."""
 
     # TODO: Add production ACF domain when known
     ALLOWED_HOSTS = ['tdp-backend-production.app.cloud.gov']
+
+    LOGIN_GOV_CLIENT_ID = os.getenv(
+        'OIDC_RP_CLIENT_ID',
+        'urn:gov:gsa:openidconnect.profiles:sp:sso:hhs:tanf-prod'
```

Comment:
We'll likely use a separate instance for production, since everything right now points to the Login.gov Identity Sandbox. At that time, we'll need to override additional Login.gov settings to reflect differing URLs, etc. However, at this time we don't have those values and this is what is configured for Prod in Login.gov.

Reply:
do we need a ticket to track this for the future? @jtwillis92

Reply:
That would depend on if we're moving forward with replacing Login.gov with Nextgen XMS. If so, this would just be part of the work for epic https://github.com/raft-tech/TANF-app/issues/1153 - specifically https://github.com/raft-tech/TANF-app/issues/1157

Otherwise, we would need a separate ticket to track Login.gov production configuration - likely as part of #721. I believe there is also some sort of contract process with Login.gov that we would have to go through to get a production tenant.